model/build.gradle

```diff
@@ -9,4 +9,7 @@ java {
 
 dependencies {
     testImplementation 'junit:junit:4.13.2'
+}
+dependencies {
+    implementation 'joda-time:joda-time:2.9.4'
 }
```

Comment:
Adding Joda is ok if there are no alternatives; however, I believe for all the applications in this PR either, `java.util.Date` or `java.time` work as well or better. Using these instead would allow us to keep the same functionality, while not needing to add Joda as a dependency. I believe this could be a good design decision.

Reply:
also minimizing dependencies is important for keeping the size of our function package low for the backend. If it gets too big, we'll have to switch to using a container image.